cmd/runtimetest/main.go

```diff
@@ -576,6 +576,28 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	t := tap.New()
+	for _, sys := range spec.Linux.Seccomp.Syscalls {
+		if sys.Action == "SCMP_ACT_ERRNO" {
+			for i, name := range sys.Names {
+				if name == "getcwd" {
+					_, err := os.Getwd()
+					if err == nil {
+						t.Diagnostic("Syscall action ERRNO can not be properly applied in the runtime")
+					}
+				} else {
+					t.Skip(i, fmt.Sprintf("%s syscall returns errno", name))
+				}
+			}
+		}
```

Comment:
And here we want an `else` case saying that we're skipping the action:

```
} else {
    t.Skip(1, fmt.Sprintf("syscall action %s", sys.Action))
}
```

or similar.